### PR TITLE
[excel] (Range) Add note about Range.getImage issue

### PR DESCRIPTION
**Pull request by AlexJerabek**
[Preview link](https://review.docs.microsoft.com/en-us/javascript/api/office-scripts/excelscript/excelscript.range?branch=AlexJ-RangeImageNote&view=office-scripts#getImage__)

Based on [this customer issue](https://docs.microsoft.com/en-us/answers/questions/547136/office-script-to-copy-table-image-copies-39wrong39.html).

### CODE REVIEW

File: docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml [hunk 1]
```diff
@@ -1124,7 +1124,12 @@ methods:
     uid: 'ExcelScript!ExcelScript.Range#getImage:member(1)'
     package: ExcelScript!
     fullName: getImage()
-    summary: Renders the range as a base64-encoded png image.
+    summary: >-
+      Renders the range as a base64-encoded png image.
+
+
+      **Note**: There is a known issue with `Range.getImage` that causes longer or wrapped text string to render without
+      line wrapping and in smaller cells. This makes the resulting image unreadable.
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml:1132]: Two parts: 
1. Based on the customer photo, it looks like this issue also affects surrounding cells – not just the cell containing the wrapped text. 
2. I think the description should offer guidance or a workaround. If there's no workaround, then perhaps something like: 
```suggestion
      line wrapping and in smaller cells. The wrapped text may merge with nearby cells, making the resulting image unreadable. Avoid using `Range.getImage` for strings longer than ___ characters and for cells with text wrapping enabled.
```

File: docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml [hunk 2]
```diff
@@ -1124,7 +1124,12 @@ methods:
     uid: 'ExcelScript!ExcelScript.Range#getImage:member(1)'
     package: ExcelScript!
     fullName: getImage()
-    summary: Renders the range as a base64-encoded png image.
+    summary: >-
+      Renders the range as a base64-encoded png image.
+
+
+      **Note**: There is a known issue with `Range.getImage` that causes longer or wrapped text string to render without
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml:1131]: Does this issue also happen with numbers?

**Review by alison-mk — COMMENTED** [review]
A question and a couple suggestions for clarification.

**Comment by AlexJerabek** [comment]
Thanks @alison-mk! I did some testing and think the latest version better describes the problem and provides a workaround. Basically, if any text exceeds the cell width (even if wrapped), you'll see this problem. This is true for both strings and numbers.

File: docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml [hunk 3]
```diff
@@ -1124,7 +1124,14 @@ methods:
...
     fullName: getImage()
-    summary: Renders the range as a base64-encoded png image.
+    summary: >-
+      Renders the range as a base64-encoded png image.
+
+
+      **Note**: There is a known issue with `Range.getImage` that causes wrapped text or text that exceeds the cell
+      width to render on the same line without line wrapping. This makes the resulting image unreadable, since the text
+      overflows across the entire row. As a workaround, make sure the data in the range fits in each of the cells as a
+      single-line.
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml:1134]: I can't find any evidence to support using a dash here, so I vote for no dash. 
```suggestion
      single line.
```

**Review by alison-mk — APPROVED** [review]
Looks great! Approved with one change that you can take or leave.

**Pull request merged**